src/runtime_src/ert/scheduler/scheduler.cpp

```diff
@@ -1229,7 +1222,7 @@ scheduler_loop()
           continue; // CU is not used
 
         auto cuvalue = read_reg(cu_idx_to_addr(slot.cu_idx));
-        if (!(cuvalue & (AP_DONE|AP_IDLE)))
```

Comment:
not sure why revert this line?

Reply:
It's a bug I introduced in previous ssv2 data flow PR 

if we don't remove AP_IDLE and we somehow do not kick start CU. 

Once ERT check it's AP_IDLE, it will notify host as a false CU completion